tests/integration/test_merge_tree_s3/test.py

```diff
@@ -1048,8 +1049,8 @@ def test_s3_engine_heavy_write_check_mem(cluster, node_name, in_flight_memory):
         "   AND type!='QueryStart'"
     ).split()
 
-    assert int(memory_usage) < 1.1 * memory
-    assert int(memory_usage) > 0.9 * memory
+    assert int(memory_usage) < 1.2 * memory
+    assert int(memory_usage) > 0.8 * memory
```

Comment:
Is it related to changes?

Reply:
I see flacks in the test. In this review. With Asan it is `1.15` instead `1.1`. So I took `1.2`.